### PR TITLE
Provide a way to test all packages using a single command #773

### PR DESCRIPTION
**Pull request by MikeRatcliffe**
Associated Issue: #773

### Summary of Changes

* Provided a way to test all packages using a single command

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,14 @@
   - [Update a Package](./docs/versioning.md#update-a-package)
   - [Updating a Launchpad Dependency](./docs/versioning.md#updating-a-launchpad-dependency)
 
+### Testing
+
+From the root folder:
+
+```js
+npm run test
```
Comment (jasonLaster) [inline README.md]: `yarn test`

we are recommending yarn because we keep yarn.lock files for consistency

**Comment by jasonLaster** [comment]
one small thing

**Comment by jasonLaster** [comment]
lets also update `circle.yml`

**Comment by MikeRatcliffe** [comment]
Tests failed because we took over 4G for one test... seems like 1 worker is probably enough.

**Comment by nchevobbe** [comment]
Looking at https://circleci.com/gh/devtools-html/devtools-core/1498?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link , it looks like we do both `npm test` and `yarn test -- --maxWorkers 2`, and `npm test` probably does not restrict the number of workers.

**Comment by MikeRatcliffe** [comment]
So let's try using npm test instead.

**Comment by MikeRatcliffe** [comment]
I have moved the maxWorkers param into package.json.

Locally `"test": "jest -- --maxWorkers=2"` does not work but `"test": "jest --maxWorkers=2"` so I have used that.

**Comment by nchevobbe** [comment]
I'd prefer having the option only on CI. locally I want the test to be as fast as possible

**Comment by nchevobbe** [comment]
I may be wrong, but I think this is gonna fail since we execute `cd ./packages/devtools-reps; yarn test --numWorkers=2`, it should be `cd ./packages/devtools-reps; yarn test -- --numWorkers=2` I think

**Comment by nchevobbe** [comment]
yep :/

**Comment by MikeRatcliffe** [comment]
It may even be `-- --numWorkers 2`... we will find the correct incantation in the end.

**Comment by MikeRatcliffe** [comment]
lol... you confused me with numWorkers but I have a good feeling about this one.

**Comment by MikeRatcliffe** [comment]
That worked but let me squash these commits.

**Comment by nchevobbe** [comment]
Yay 🎉

**Pull request merged**